packages-proprietary/tokens/src/components/ams/page-header.tokens.json

```diff
@@ -24,6 +24,7 @@
         "text-wrap": { "value": "{ams.typography.heading.text-wrap}" }
       },
       "mega-menu": {
+        "padding-block": { "value": "{ams.space.xl}" },
```

Comment:
This results in 92 pixels of bottom padding (at 1600 wide): `xl` from mega menu + `lg` from Header = 56 + 36. @Bondt007 says this would be fine, but we could also set `padding-block-end: calc(var(--ams-space-xl) - var(--ams-space-lg))` to get 84 pixels, which is the intended `--ams-space-2xl`.

Reply:
This is not the solution I was expecting, if you look at the story there is a Grid component in the Page Header:

```jsx
<Grid gapVertical="small" paddingBottom="large">
  <PageHeader.GridCellNarrowWindowOnly span="all">
    <LinkList>
```

If you change it to 

```jsx
<Grid gapVertical="small" paddingVertical="large">
  <PageHeader.GridCellNarrowWindowOnly span="all">
    <LinkList>
```

It fixes the problem and you still allow internal apps to reduce the padding here without overrides. There is also a variant where the mega menu has a heading ("Alle onderwerpen") which is also possible because you can reduce the padding on the top. Heading line height + gap + padding would be a bit much.